### PR TITLE
chore(release-notes): Updating release notes 2.32.0 with known issues

### PR DESCRIPTION
**Pull request by christosarvanitis**
Resolves GitHub issue: <link to GH issue>

Resolves Jira: [JIRAPROJECT-123]

### CODE REVIEW

File: content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-32-0.md [hunk 1]
```diff
@@ -91,6 +91,15 @@ Related OSS PRs:
 
 ## Known issues
 <!-- Copy/paste known issues from the previous version if they're not fixed. Add new ones from OSS and Armory. If there aren't any issues, state that so readers don't think we forgot to fill out this section. -->
+### Terraform pipelines S3 Artifact Store does not support AuthZ applications
```
Comment (jasonmcintosh) [inline content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-32-0.md]: ## When using S3 artifact stores and Application roles, Terraformer will fail to fetch artifacts

Comment (jasonmcintosh) [reply content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-32-0.md]: Probably could use some other tweaking, as it is right now... it doesn't make a lot of sense

Comment (christosarvanitis) [reply content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-32-0.md]: Yeah it doesnt. How about now?

Comment (jasonmcintosh) [reply content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-32-0.md]: With S3 artifact storage enabled Terraformer stages break when authentication is set on applications

Comment (christosarvanitis) [reply content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-32-0.md]: done

**Review by jasonmcintosh — APPROVED** [review]

**Review by kkotula — APPROVED** [review]

**Pull request merged**